owncloudApp/src/main/java/com/owncloud/android/ui/dialog/FileAlreadyExistsDialog.kt

```diff
@@ -52,18 +55,22 @@ class FileAlreadyExistsDialog private constructor(
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        binding.dialogFileAlreadyExistsTitle.text = titleText
-        binding.dialogFileAlreadyExistsInformation.text = descriptionText
-        binding.dialogFileAlreadyExistsCheckbox.text = checkboxText
-        binding.dialogFileAlreadyExistsKeepBoth.setOnClickListener { dialogClickListener!!.onKeepBothButtonClick() }
-        binding.dialogFileAlreadyExistsCheckbox.setOnCheckedChangeListener { _, isChecked ->
-            isCheckBoxChecked = isChecked
-        }
-        binding.dialogFileAlreadyExistsReplace.setOnClickListener { dialogClickListener!!.onReplaceButtonClick() }
-        binding.dialogFileAlreadyExistsSkip.setOnClickListener { dialogClickListener!!.onSkipButtonClick() }
+        binding.let { binding ->
```

Comment:
You can remove the let, because you add a '?' after the dialogClickListener.

Reply:
Done